validator/testcases/javascript/actions.py

```diff
@@ -643,33 +651,36 @@ def _expr_assignment(traverser, node):
                     # If it's a global and the actual member exists, test
                     # whether it can be safely overwritten.
                     member = member_object_value["value"][member_property]
-                    readonly_value = (member["readonly"] if
-                                      "readonly" in member else
-                                      True)
+                    if "readonly" in member:
+                        global_overwrite = True
+                        readonly_value = member["readonly"]
 
         traverser._debug("ASSIGNMENT:DIRECT:GLOB_OVERWRITE %s" %
-                             global_overwrite)
+                         global_overwrite)
+        traverser._debug("ASSIGNMENT:DIRECT:READONLY %r" %
+                         readonly_value)
 
-        if (global_overwrite and
-            not traverser.is_jsm and
-            not traverser.err.get_resource("em:bootstrap") and
-            readonly_value == True):
+        if callable(readonly_value):
+            # The readonly attribute supports a lambda function that accepts
```

Comment:
On Tue, May 05, 2015 at 07:58:18AM -0700, Mathieu Agopian wrote:
>>
>> -        if (global_overwrite and
>> -            not traverser.is_jsm and
>> -            not traverser.err.get_resource("em:bootstrap") and
>> -            readonly_value == True):
>> +        if callable(readonly_value):
>> +            # The readonly attribute supports a lambda function that accepts
>
>It's missing the end of the sentence?

That it is. Unfortunately, I'm not entirely sure what it's 
supposed to be. I didn't actually change this line, just the 
code around it.